### PR TITLE
release-notes: updates for 20230110 releases

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: release-notes/stable.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 releases:
+  37.20221225.3.0:
+    issues:
+      - text: "add back in criu-libs package"
```
Comment (jlebon) [inline release-notes/stable.yml:4]: s/add/Add/ for consistency?

**Review by jlebon — APPROVED** [review]
One nit, otherwise LGTM!

File: release-notes/stable.yml (hunk 1 above)
Comment (dustymabe) [reply release-notes/stable.yml]: fixed

**Review by marmijo — APPROVED** [review]
LGTM

**Review by jlebon — APPROVED** [review]

**Pull request merged**